search-app/src/cmr/search/system.clj

```diff
@@ -55,7 +79,11 @@
   "App logging level"
   {:default "info"})
 
-(def search-public-conf
+(def public-conf
```

Comment:
Oh I thought this needed to be a function in order to override with configuration from environment variables at runtime. Now I'm confused 😕